openr/docs/Protocol_Guide/KvStore.md

```diff
@@ -80,7 +80,7 @@ getKvStoreKeyVals(std::string area, thrift::KeyGetParams keyGetParams)
  * @return: None
  */
 void
-setKvStoreKeyVals(std::string area, thrift::KeyGetParams keyGetParams)
+setKvStoreKeyVals(std::string area, thrift::KeySetParams keySetParams)
```

Comment:
```suggestion
setKvStoreKeyVals(std::string area, thrift::KeySetParams KeySetParams)
```

Is this a typo in the typo fix?

Reply:
For parameter names, we are following camel naming format, and first character is lower case? Thus, the fix does not have typo?

Reply:
Sorry, yeah my bad, I thought this was two types.